.github/ISSUE_TEMPLATE/documentation_request_correction.yml

```diff
@@ -62,9 +62,9 @@ body:
     id: terms
     attributes:
       label: Code of Conduct
-      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/NVIDIA/AIQToolkit/blob/develop/CODE-OF-CONDUCT.md)
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/NVIDIA/AIQtoolkit/blob/develop/CODE-OF-CONDUCT.md)
```

Comment:
The github links automatically redirect/resolve to `AIQToolkit` -- I'm not sure if changing the URLs makes sense since it would always force an HTTP redirect.